prometheus/src/main/java/org/opensearch/sql/prometheus/client/PrometheusClientImpl.java

```diff
@@ -83,6 +83,17 @@ public Map<String, List<MetricMetadata>> getAllMetrics() throws IOException {
     return new ObjectMapper().readValue(jsonObject.getJSONObject("data").toString(), typeRef);
   }
 
+  @Override
+  public List<String> getAllSeriesLabels() throws IOException {
```

Comment:
Can you add use case to the PR description, what are we trying to achieve?

Reply:
Updated.  Please review.

Reply:
I'm only looking for your review on throughput and generic functionality.
There are some hardcoded values that I will produce dynamically in a fast-follow commit on this PR